### PR TITLE
refactor(utils/misc): add human friendly duration utilities

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

File: routes/caniuse/index.js [hunk 1]
```diff
@@ -40,7 +41,6 @@ async function route(req, res) {
     return;
   }
 
-  // cache for 24hours (86400 seconds)
-  res.set("Cache-Control", "max-age=86400");
+  res.set("Cache-Control", `max-age=${seconds("24h")}`);
```
Comment (sidvishnoi) [inline routes/caniuse/index.js:44]: I don't have a strong opinion, but would the following be nicer?
``` js
  res.set("Cache-Control", `max-age=${seconds`24h`}`);
```

File: utils/misc.js [hunk 2]
```diff
@@ -9,6 +9,57 @@ function env(name) {
...
+/**
+ * Convert a human readable duration string to seconds value.
+ * @param {string} duration
+ * ``` js
+ * seconds("1m") // 60
+ * seconds("1.5m") // 90
+ * ```
+ */
+function seconds(duration) {
+  if (typeof duration === "number") {
```
Comment (marcoscaceres) [inline utils/misc.js:29]: nit: Marcos says, "looks a lot like a switch statement" ;)

File: routes/caniuse/index.js (hunk 1 above)
Comment (marcoscaceres) [reply routes/caniuse/index.js:44]: Function is fine.

File: utils/misc.js (hunk 2 above)
Comment (sidvishnoi) [reply utils/misc.js:29]: I'm going to remove support for number as parameter type instead :wink:

**Pull request merged**